### PR TITLE
Channel and Playlist Suggestion Form

### PR DESCRIPTION
**Pull request by craiglonsdale**
[Add a Channel](https://deploy-preview-52--breadtubetv.netlify.com/channels/new/)
[Add a Playlist](https://deploy-preview-52--breadtubetv.netlify.com/playlists/new/)

### CODE REVIEW

**Comment by craiglonsdale** [comment]
This needs a front end pass, because it looks horrible. My CSS skills would just break everything. Happy for someone to take a run at it.

**Comment by dirkkelly** [comment]
Pushed some design updates @craiglonsdale, put it up for review if you're happy with it. We'll deploy this one together so we can configure Netlify.

**Comment by dirkkelly** [comment]
@craiglonsdale Think we should make the urls a textarea and just allow new lines

File: layouts/partials/playlists/new.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="flex-fill">
+  <form name="add_playlist" method="POST" data-netlify="true" netlify-honeypot="important-field">
```
Comment (dirkkelly) [inline layouts/partials/playlists/new.html]: ## Confirmed Submission

screen shot 2019-01-30 at 4 38 49 pm

## Confirmed Email

screen shot 2019-01-30 at 4 37 19 pm

**Pull request merged**